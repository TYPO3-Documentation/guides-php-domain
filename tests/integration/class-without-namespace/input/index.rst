=========
PHP Class
=========

..  php:class:: TestClass

    Lorem Ipsum Dolor!
