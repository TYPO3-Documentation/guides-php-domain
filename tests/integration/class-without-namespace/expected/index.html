<!-- content start -->
    <div class="section" id="php-class">
            <h1>PHP Class</h1>
            <dl class="php class">
    <dt class="sig sig-object php" id="testclass">
        <em class="property"><span class="pre">class</span> </em>
        <span class="sig-name descname"><span class="pre">TestClass</span></span> <a class="headerlink" href="#testclass" title="Permalink to this definition">Â¶</a>
    </dt>
    <dd>

    <p>Lorem Ipsum Dolor!</p>

    </dd>
</dl>
    </div>
<!-- content end -->
